### ISSUE TITLE
Error: failed to prune cache "assets"

### ISSUE THREAD

**Issue by rahilwazir** [gohugoio/hugo#5745]
I'm using the following theme: https://github.com/rhazdon/hugo-theme-hello-friend-ng.git

And when I try to deploy it on Netlify, I face the following error.

```
> hugo --gc --minify
Total in 27 ms
Error: failed to prune cache "assets": remove /home/rw/html/rahilw.dev/resources/_gen/assets/js/js/main.js_d11fe7b62c27961c87ecd0f2490357b9.content: no such file or directory
```

This also happens on my local machine.

**Comment by bep** [comment]
Pretty sure this is the same as #5673

**Comment by rahilwazir** [comment]
I'm not sure but to give you more detail..

**Terminal**
![terminal](https://user-images.githubusercontent.com/1818941/54077847-f3fb8c80-42df-11e9-821b-9574e6c5930c.png)

**File Tree**

![sidebar](https://user-images.githubusercontent.com/1818941/54077846-f1009c00-42df-11e9-9f63-3fd63768f136.png)

There are two nested `js` and `scss` directories, Is it intentional?

Let me know if it still the same as #5673

**Comment by neslinesli93** [comment]
@rahilwazir did you find the bug?

**Comment by rahilwazir** [comment]
Nah man, I changed my theme and it worked out for me. Are you using the theme?

**Comment by neslinesli93** [comment]
Yep, however I went and used `hello-friend`, and it works like a charm. I'd like to use this one because the defaults look way better...

**Comment by neslinesli93** [comment]
Unfortunately we can't open issues on `hello-friend-ng` repo, so I'm pinging @rhazdon and hope he sees it

**Comment by rhazdon** [comment]
@neslinesli93, thanks for pinging. I will take a look at it. I've also enabled the issue tracker for the repo...

**Comment by neslinesli93** [comment]
Thanks man, I was sending you an email regarding the issue, but thankfully you enabled issues...

**Comment by rhazdon** [comment]
@neslinesli93, @rahilwazir 

I checked the issue and found the problem. The "bug" is the `resources` folder in the theme itself. Usually, you don't need this folder, so feel free and delete it. Then it should work as expected. 

Hugo asked me to add the folder to the repo https://github.com/gohugoio/hugoThemes/issues/577#issuecomment-466491704, because `hello-friend-ng` uses the Hugo asset pipelines instead of webpack (or similar tools). That means that your assets are genreated completly fresh when you compile your project. 
Netlify had some issues with this pipeline stuff... I don't know if it is already fixed.

**Comment by neslinesli93** [comment]
How can we do this by keeping yout theme as a submodule?

**Comment by rhazdon** [comment]
@digitalcraftsman, do you know maybe, if the issue regarding the hugo asset pipeline and the theme page is solved?

**Comment by rhazdon** [comment]
@digitalcraftsman Sorry for tagging you again. Are there any news regarding Netlify? Is it possible to delete the `resources` folder from the repo?

**Comment by THeK3nger** [comment]
Thehttps://github.com/htr3n/hyde-hyde theme has the same problem.

**Comment by jzeneto** [comment]
The [Hermit theme](https://themes.gohugo.io/hermit/) has the same problem (as @kimfucious and I have reported [here](https://discourse.gohugo.io/t/hugo-build-error-with-gc-minify/20600/13?u=jzeneto)).
@Track3

**Comment by jpcaruana** [comment]
For me force removing `_gen` folders before runnning hugo works fine, and build is ok on netlify

In my `netlify.toml`:

`command = "find . -name _gen -exec rm -rf {} + ; hugo --gc --minify"`

Hope this helps

**Comment by hayorov** [comment]
> For me force removing `_gen` folders before runnning hugo works fine, and build is ok on netlify
> 
> In my `netlify.toml`:
> 
> `command = "find . -name _gen -exec rm -rf {} + ; hugo --gc --minify"`
> 
> Hope this helps

Reasonable workaround. Thanks.
